src/components/PathwaysList/NewPathwayModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useCallback, useRef, memo, FormEvent } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faTimes } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+  IconButton
+} from '@material-ui/core';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
This is super minor but I know in other places we use shortid to generate ids. I feel like we should only use one just to be consistent and so we are not importing multiple things to do the same job

Reply:
Oops I didn't see we already had an id generator. Fixed!